Makefile

```diff
@@ -29,10 +29,11 @@ stop:
 
 # Run tests for the library
 test:
-	poetry export -f requirements.txt --without-hashes --with test --output requirements.txt
-	docker compose -f docker-compose.dev.yml up -d --build --wait
-	docker compose exec -T backend pytest --cov=app
-	docker compose -f docker-compose.dev.yml down
+	-poetry export -f requirements.txt --without-hashes --with test --output requirements.txt
```

Comment:
btw, why are there leading hyphens here?

Reply:
leading hyphens are really useful to launch the "docker compose down" even if there are any issue in the commands before

Reply:
Oh thanks for teaching me, I felt like I didn't something there!
Mind opening a PR after that one to adjust this on all relevant make commands?

Reply:
yes ok no pb !